src/helpers/IntentBuilder.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: BUSL-1.1
+// This code is made available under the terms and conditions of the Business Source License 1.1 (BUSL-1.1).
+// The act of publishing this code is driven by the aim to promote transparency and facilitate its utilization for educational purposes.
+
+pragma solidity 0.8.18;
+
+import { WordCodec } from "@hmx/libraries/WordCodec.sol";
+import { ConfigStorage } from "@hmx/storages/ConfigStorage.sol";
+
+contract IntentBuilder {
```

Comment:
Is this suppose to be testing only?

Reply:
Not recommended frontend to call this when building the intent. Should be done locally.

Reply:
Then please move it under `tests` folder